manifests/a/Ablaze/Floorp/10.14.0/Ablaze.Floorp.locale.en-US.yaml

```
@@ -0,0 +1,27 @@
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.version.1.4.0.schema.json
+PackageIdentifier: Ablaze.Floorp
+PackageVersion: 10.14.0
+PackageLocale: en-US
+Publisher: Ablaze
+PublisherUrl: https://ablaze.one/
+PublisherSupportUrl: https://support.ablaze.one/articles/floorp/
+PrivacyUrl: https://docs.ablaze.one/floorp_privacy_policy/
+Author: Ablaze
+PackageName: Ablaze Floorp
+PackageUrl: https://floorp.app/
+License: Mozilla Public License Version 2.0
+LicenseUrl: https://www.mozilla.org/en-US/MPL/2.0/

```

```suggestion
LicenseUrl: https://github.com/Floorp-Projects/Floorp/blob/ESR102/LICENSE
```
In LicenseUrl, we usually link to where you can find information about a package's license.